cmd/crc/cmd/delete.go

```diff
@@ -87,3 +93,49 @@ func (s *deleteResult) prettyPrintTo(writer io.Writer) error {
 	}
 	return nil
 }
+
+// TODO: remove it after we do one more upgrade of the podman bundle from 4.4.1
+func removeCRCHostEntriesFromKnownHosts() error {
+	knownHostsPath := filepath.Join(constants.GetHomeDir(), ".ssh", "known_hosts")
+	f, err := os.Open(knownHostsPath)
+	if err != nil {
+		return fmt.Errorf("Unable to open user's 'known_hosts' file: %w", err)
+	}
+	defer f.Close()
+
+	tempHostsFile, err := os.CreateTemp("", "crc")
+	if err != nil {
+		return fmt.Errorf("Unable to create temp file: %w", err)
+	}
+	defer func() {
+		os.Remove(tempHostsFile.Name())
+		tempHostsFile.Close()
+	}()
+
+	if err := tempHostsFile.Chmod(0600); err != nil {
+		return fmt.Errorf("Error trying to change permissions for temp file: %w", err)
+	}
+
+	var foundCRCEntries bool
+	scanner := bufio.NewScanner(f)
+	writer := bufio.NewWriter(tempHostsFile)
+	for scanner.Scan() {
+		if strings.Contains(scanner.Text(), "[127.0.0.1]:2222") {
+			foundCRCEntries = true
+			continue
```

Comment:
This is only for user networking mode we have to consider system network mode in case of Linux

Reply:
yes but since system-mode networking is only on linux, and there i think less people should be using the podman preset.. and after we upgrade podman bundle again from 4.4.1 to the next version this will not be required even for user-mode networking

Reply:
I don't think it is fixed on 4.4.1 side also so we do need to have a long term solution for it.

Reply:
updated, it should remove system-mode entries also now